core/src/main/scala/org/http4s/util/io/package.scala

```diff
@@ -1,91 +0,0 @@
-/** TODO fs2 port
```

Comment:
Why are we dropping this? Not that we shouldn't just checking.

Reply:
This would be https://github.com/functional-streams-for-scala/fs2/blob/series/0.10/io/src/main/scala/fs2/io/io.scala instead, i think.